tests/discontinuous_composition_bound_preserving_limiter.prm

```diff
@@ -0,0 +1,103 @@
+# A description of the falling box benchmark see the reference:
+# Gerya, T. V., Yuen, D. A., 2003a. Characteristics-based marker-in-cell method
+# with conservative finite-differences schemes for modeling geological flows with
+# strongly variable transport properties
```

Comment:
Please add a sentence here that says that this test is specifically designed for the bound-preserving limiter. That makes it easier to figure it out (I know the test is already named like that, but this is the standard procedure :smile:)